src/lf_apply.rs

```diff
@@ -88,23 +84,22 @@ unsafe fn backup_lpf<BD: BitDepth>(
     if lr_backup != 0 && frame_hdr.size.width[0] != frame_hdr.size.width[1] {
         while row + stripe_h <= row_h {
             let n_lines = 4 - (row + stripe_h + 1 == h) as c_int;
-            let mut dst_guard = dst.mut_slice_as(dst_offset..dst_offset + dst_w as usize);
+            let mut dst_guard = dst.mut_slice_as((dst_offset.., ..dst_w));
             dsp.mc.resize.call::<BD>(
                 dst_guard.as_mut_ptr(),
                 dst_stride,
-                src.as_ptr().add(src_offset),
-                src_stride,
-                dst_w,
+                src.data.as_ptr_at::<BD>(src.offset),
```

Comment:
Another common pattern I'm seeing is `foo.data.slice((foo.offset.., ..len))`. Would it make sense to add something like a `slice_n` helper so this could be simplified to `foo.slice_n(len)`?

Reply:
Yeah, maybe, I've been considering that.  I might add it as a PR on the top of the stack.